common/templates/collection-as-table.njk

```diff
@@ -30,4 +30,24 @@
       }
     }) }}
   {% endif %}
+
+  {% if resultsAsTable.cancelled.rows.length %}
+    {% set html %}
+      <div>
+        {{ govukTable({
+          rows: resultsAsTable.cancelled.rows,
+          head: resultsAsTable.cancelled.head,
+          classes: "govuk-secondary-text-colour"
+        }) }}
+      </div>
+    {% endset %}
+    {{ govukDetails({
```

Comment:
Maybe add a bit of breathing room here. It can help with scanning to use a new line between blocks when they are at the same indentation.